Kiosk/Admin/AuctionWebViewController.swift

```diff
@@ -18,6 +27,6 @@ class AuctionWebViewController: WebViewController {
             _ = self?.navigationController?.popViewController(animated: true)
             return
         }
-        self.present(passwordVC, animated: true) {}
+        self.present(passwordVC, animated: true) { }
```

Comment:
In JS we'd not put a space here.

Reply:
Right now I deliberately add one to make the "do nothing" more visible. In JS you have have more around it: `function(){}` or `() => {}`.

Don't have a strong preference on this though.

@orta What's your take?

Reply:
I also don't have a strong preference - had I not see it I'd have gone with no space, but I kinda like the space now that I see it

Reply:
The space "feels" more "Swift" to me, but I don't know why.